fxdc/fields.py

```
@@ -0,0 +1,81 @@
+            if len(desc) > 50:
+                raise FieldError("Description Should Not Be More Than 50 Characters")
+
+        self.__verbose_name = verbose_name
+        self.__default = default
+        self.__typechecking = typechecking
+        self.__null = null
+        self.__blank = blank
+        self.__desc = desc
+
+    @property
+    def desc(self) -> Optional[str]:
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)